rust/macros/helpers.rs

```diff
@@ -28,6 +28,21 @@ pub(crate) fn try_byte_string(it: &mut token_stream::IntoIter) -> Option<String>
     })
 }
 
+pub(crate) fn try_string(it: &mut token_stream::IntoIter) -> Option<String> {
+    try_literal(it).and_then(|string| {
+        // TODO: handle raw string literals
```

Comment:
Maybe as documentation of the function, to avoid the `TODO` (if we do not expect tot need raw literals).

Or maybe panic, if you can detect it is a raw literal?